.gitignore

```diff
@@ -8,3 +8,7 @@ server/node_modules
 # next build output
 client/.next
 
+.DS_Store
+server/.serverless/sls-cms-server-app.zip
+server/.serverless/**
+server/config/remoteEnv.yml
```

Comment:
this file needs to get created + filled in with correct env vars for lambda to work on AWS